src/main/java/io/github/ImpactDevelopment/installer/target/targets/Vanilla.java

```diff
@@ -190,6 +201,7 @@ public void install(boolean allowMinecraftToBeOpen) throws IOException {
         sanityCheck(allowMinecraftToBeOpen);
         installVersionJson();
         installProfiles();
+        installOptifine();
```

Comment:
Is this compatible with how Impact calls the installer from its internal updater? I strongly don't think it will be lol

Reply:
good point. Trying to think of a good way to specify that optifine is _already_ installed.